### DISCUSSION TITLE
怎么加入新的量化模型

### DISCUSSION THREAD

**Discussion by mengxingGG** [GaiZhenbiao/ChuanhuChatGPT#968] [Q&A] [answered]
不能只等作者更新，这会增加不少工作负担，也许可以做出指引可以让使用者自行加入部署新的模型。比如量化版的Qwen72b，以及量化后的Qwen14b，或者微调版本的chatGPT3.5等其他可用模型。

**Comment by GaiZhenbiao** [comment, marked as answer]
1. 将模型权重放在models文件夹下（软链接也行），会自动加载。
2. 可以用内置的GPT训练功能获取finetune的GPT模型，并加入到模型列表里。
